std/file.d

```diff
@@ -2983,6 +2983,31 @@ auto dirEntries(string path, SpanMode mode, bool followSymlink = true)
     return DirIterator(path, mode, followSymlink);
 }
 
+/// Duplicate functionality of D1's $(D std.file.listdir()):
+unittest
+{
+  string[] listdir(string pathname)
+  {
+    import std.file;
+    import std.path;
```

Comment:
What JakobOvrum said. We are trying to phase out string lambdas, for anything less trivial than basic arithmetic. If it requires any import, then "normal" lambdas is what we are trying to promote. In particular, since this entire block is meant to be documentation, I really think it should be using normal lambdas.